logstash-core/lib/logstash/api/commands/node.rb

```diff
@@ -27,7 +27,7 @@ def pipelines
         def pipeline(pipeline_id)
           extract_metrics(
             [:stats, :pipelines, pipeline_id.to_sym, :config],
-            :workers, :batch_size, :batch_delay, :config_reload_automatic, :config_reload_interval, :dead_letter_queue_enabled, :dead_letter_queue_path
+            :graph, :ephemeral_id, :hash, :workers, :batch_size, :batch_delay, :config_reload_automatic, :config_reload_interval, :dead_letter_queue_enabled, :dead_letter_queue_path
```

Comment:
Nitpick: even though it's not the way the code is now, I find these array-of-symbol type arguments to be best listed one per line, to make future additions and removals clearer. As it stands, we're flowing pretty far off the right margin, which increases the difficulty of a developer understanding what all is going on.

Reply:
Totally agree. Great point. Will fix.